docs/add-ins/use-rest-api.md

```diff
@@ -110,4 +110,4 @@ function getCurrentItem(accessToken) {
 ## See also
 
 - For an example that calls the REST APIs from an Outlook add-in, see [command-demo](https://github.com/OfficeDev/outlook-add-in-command-demo) on GitHub.
-- Outlook REST APIs are also available through the Microsoft Graph endpoint (see [Outlook REST API via Microsoft Graph](../rest/index.md#outlook-rest-api-via-microsoft-graph)) but there are some key differences, including how your add-in gets an access token. For more details, please [Compare Microsoft Graph and Outlook](../rest/compare-graph.md).
\ No newline at end of file
+- Outlook REST APIs are also available through the Microsoft Graph endpoint but there are some key differences, including how your add-in gets an access token. For more details, please [Compare Microsoft Graph and Outlook](../rest/compare-graph.md).
```

Comment:
For some reason your changes don't seem to reflect the contents of my 'suggestion' that I added in prior review. So let's do this the old fashioned way -- suggest that you update this list item to be:

`- Outlook REST APIs are also available through the Microsoft Graph endpoint. For more information, see [Outlook REST API via Microsoft Graph](../rest/index.md#outlook-rest-api-via-microsoft-graph).`

Reply:
I've updated the link but I wanted to include the additional info about the differences because that's what led to this issue being raised.